os-autoinst-obs-auto-submit

```diff
@@ -173,37 +194,63 @@ handle_auto_submit() {
     )
 }
 
-prefix="${prefix:-""}"
-[ "$dry_run" = "1" ] && prefix="echo"
-osc="${osc:-"$prefix retry -e -- osc"}"
+get_project_packages() {
+    echo "${packages:-$($osc ls "$dst_project" | grep -v '\-test$')}"
+}
 
-# submit from staging project if specified
-[[ $staging_project && $staging_project != none && $staging_project != "$src_project" ]] && src_project=$staging_project from_staging=1
+has_pending_submission() {
+    # throttle number of submissions to allow only one SR within a certain number of days
+    # note: Avoid using `grep --quiet` here to keep consuming input so osc does not run into "BrokenPipeError: [Errno 32] Broken pipe".
+    if [[ $throttle_days != 0 ]] && $osc request list --project "$dst_project" --type submit --state new,review --mine --days "$throttle_days" | grep 'Created by' > /dev/null; then
+        echo "Skipping submission, there is still a pending SR younger than $throttle_days days."
+        return 1
+    fi
+    return 0
+}
 
-if [[ -e job_post_skip_submission ]]; then
-    echo "Skipping submission, reason: "
-    cat job_post_skip_submission
-    exit 0
-fi
+is_staging_specified() {
+    if [[ -n $staging_project && $staging_project != none && $staging_project != "$src_project" ]]; then
+        src_project="$staging_project"
```

Comment:
I think this is causing a regression, overriding the `src_project` now does happen only for the clean-up job (`osc rdelete`) and not before `handle_auto_submit` anymore so the update is always from the devel project and not from the staging project. Am I correct?

Reply:
https://github.com/os-autoinst/scripts/pull/403